docs/content/preview/reference/configuration/yb-master.md

```diff
@@ -158,6 +158,27 @@ Monitoring web server home.
 
 Default: The `www` directory in the YugabyteDB home directory.
 
+##### --webserver_certificate_file
+
+Location of the SSL certificate file (in .pem format) to use for the web server. If empty, SSL is
+not enabled for the web server.
+
+Default: `""`
```

Comment:
Probably clearer to say this, rather than some quotation marks. If you agree, do for all. :)
```suggestion
No default value.
```

Reply:
No default value sounds to me like it's unset (and therefore is required to be set), versus set to a placeholder value, which does not have to be overridden.